ui/raidboss/data/06-ew/raid/p1s.ts

```diff
@@ -176,30 +231,70 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
-    // Copy/paste from normal, seems to be the same
     {
       id: 'P1S Powerful Light/Fire',
       type: 'GainsEffect',
       netRegex: NetRegexes.gainsEffect({ effectId: '893', capture: true }),
-      alertText: (_data, matches, _output) => {
-        if (matches.count === '14C')
-          return _output.light!();
-        return _output.fire!();
+      preRun: (data, matches) => {
+        if (matches.count === '14C') {
+          data.safeColor = 'light';
+          return;
+        }
+        data.safeColor = 'fire';
       },
+      alertText: (data, _matches, output) => data.safeColor && output[data.safeColor]!(),
+      outputStrings: fireLightOutputStrings,
+    },
+    {
+      id: 'P1S Shackles of Time on You',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AB5' }),
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.text!(),
+      // For raid calling purposes, this might be useful but don't clutter TTS.
```

Comment:
Maybe this should always call the person it's on with info text? Shackles on X, Shackles on YOU sorta thing?